ecommerce/settings/appsembler.py

```diff
@@ -0,0 +1,16 @@
+"""Appsembler settings"""
+
+from ecommerce.settings.production import *
+
+
+# SENTRY Integration
+SENTRY_DSN = AUTH_TOKENS.get('SENTRY_DSN', False)
```

Comment:
`SENTRY_DSN = AUTH_TOKENS.get('SENTRY_DSN', False)` -> `SENTRY_DSN = AUTH_TOKENS.get('SENTRY_DSN')`

`dict.get()` defaults to `None` and `None` is a falsy value.

Reply:
This is something we should probably hash out in the style guide PR instead, but I tend to prefer an explicit two argument `dict.get()` because it forces one to think about the alternate case and what the "not found" value and code path really should be. Otherwise, I've seen programmers just habitually replace `dict[k]` with `dict.get(k)` and then somewhere further away the code gets an unexpected `None` that takes more work to track down.

Anyway, that's just my opinion. But I also like programming in Go, so clearly there's something wrong with me. :)

Reply:
Both `dict.get(something)` and `dict.get(something, None)` seems OK to me, with preference to the former more succinct version.

According to this not super scientife method of reasearch, it seems that there's also a consensus about `dict.get(something)`:

 - StackOverflow: [Return None if Dictionary key is not available](https://stackoverflow.com/questions/6130768/return-none-if-dictionary-key-is-not-available)

Reply:
Thanks @OmarIthawi and @thraxil , yes Omar i was thinking the same about `dict.get(something)` method it may cause some issue in feature if we didn't provide that key in vars